resources/email_templates/once_the_speaker_confirms_for_the_talk.md

```diff
@@ -0,0 +1,24 @@
+Hello,
+
+Thank you for confirming the workshop/talk/hands-on. Your session will truly benefit the group.
+
+Now to plan the session can you please provide the following information, whichever you are okay in sharing in with public.
```

Comment:
```suggestion
We require some information to plan this session and publicly promote your session. You do not have to provide any information that isn't relevant or that you don't want to share. However it would help if you could give us as many details as possible.
```